sljit_src/sljitNativeLOONGARCH_64.c

```diff
@@ -3690,7 +3695,10 @@ SLJIT_API_FUNC_ATTRIBUTE sljit_s32 sljit_emit_atomic_store(struct sljit_compiler
 	CHECK_ERROR();
 	CHECK(check_sljit_emit_atomic_store(compiler, op, src_reg, mem_reg, temp_reg));
 
-	if (!(LOONGARCH_CFG2_LAMCAS & get_cpu_features(GET_CFG2))) {
+	if ((op & SLJIT_ATOMIC_USE_CAS) && !LOONGARCH_SUPPORT_AMCAS)
```

Comment:
`LOONGARCH_SUPPORT_AMCAS` evaluated twice. `if (op & SLJIT_ATOMIC_USE_CAS) ..` would be moved into the if below, then one evaluation is enough.

Reply:
Fixed, thanks.